doc/rst/language/spec/interoperability.rst

```diff
@@ -544,8 +542,8 @@ type    char\*
 Currently, ``param`` arguments are not allowed in an extern function
 declaration, and ``type`` args are passed as a string containing the
 name of the actual type being passed. Note that the level of indirection
-is changed when passing arguments to a C function using ``inout``,
-``out``, or ``ref`` intent. The C code implementing that function must
+is changed when passing arguments to a C function using
+``ref`` intent. The C code implementing that function must
```

Comment:
Should we have a "the" here?

Reply:
Not sure which "the" you are talking about or if you wanted me to add or remove it

Reply:
Sorry, I was thinking as in "the ref intent"